njsscan/__main__.py

```diff
@@ -84,6 +84,65 @@ def json_output(outfile, scan_results):
         return json_output
 
 
+def sonarqube_output(outfile, scan_results):
+    """Sonarqube JSON Output."""
+    sonarqube_issues = []
+    for k, v in scan_results['nodejs'].items():
+        issue = get_sonarqube_issue(v)
+        issue['ruleId'] = k
+        sonarqube_issues.append(issue)
+    sonarqube_report = {
+        'issues': sonarqube_issues,
+    }
+    if outfile:
+        with open(outfile, 'w') as of:
+            json.dump(sonarqube_report, of, sort_keys=True,
+                      indent=2, separators=(',', ': '))
+    else:
+        json_output = (json.dumps(sonarqube_report, sort_keys=True,
+                                  indent=2, separators=(',', ': ')))
+        print(json_output)
+        return json_output
```

Comment:
This section is redundant. You could simply call `json_output(outfile, sonarqube_report)`